site/src/app/layout.tsx

```diff
@@ -8,7 +8,7 @@ import type { Metadata } from 'next';
 import './app.css';
 
 export const metadata: Metadata = {
-    title: 'nRF Connect SDK App Index',
+    title: 'nRF Connect SDK Add-On Index',
     description: 'A live index of applications and samples available for the nRF Connect SDK.',
```

Comment:
```suggestion
    description: 'A live index of supplementary components available for the nRF Connect SDK.',
```
I understand from the Teams chat it's not only about apps and samples? But if our index only lists apps and samples, disregard my comment.